docs/source/upgrade_notes.rst

```diff
@@ -43,6 +43,11 @@ Upgrade Notes
   See :doc:`purging old data documentation <troubleshooting/purging_old_data>` for further
   information.
 
+* As part of extending RBAC support to include protecting access to datastore operations, if
+  you have RBAC enabled and any sensors access the datastore, then the ``sensor_service`` will need to
```

Comment:
Is sensor_service the user?

Reply:
Yes, updated text to make explicit.

Reply:
Looks good to me and should address the RBAC access issues with sensors going forward.  Is there a corresponding entry that mentions this or the "sensor_service" user in the RBAC docs for users that have already updated to 3.7 but are just starting to enable RBAC?

Reply:
I couldn't find a mention! I've just updated the PR to add sensor_service to the rbac page - hope that addresses it.

Reply:
Thanks. Looks good.